frigg_worker/cli.py

```diff
@@ -23,17 +17,36 @@ def load_logging_config():
 @click.option('--hq-url', default=None, help='URl for frigg-hq, overrides settings')
 @click.option('--hq-token', default=None, help='Token for frigg-hq, overrides settings')
 @click.option('--slack-url', default=None, help='URL for incoming webhook in slack')
-def start(**kwargs):
-    load_logging_config()
+@click.option('--loglevel', default='DEBUG', help='Set log level for frigg-packages')
+def start(**options):
+    options = evaluate_options(options)
+    logging.config.dictConfig(load_logging_config(options))
 
     try:
-        print("Starting frigg worker")
-        fetcher(**kwargs)
+        logger.info('Starting frigg worker')
+        fetcher(**options)
     except Exception as e:
-        print(e)
         logger.error(e)
         sentry.captureException()
 
 
+def evaluate_options(options):
+    if options['dispatcher_url'] is None:
```

Comment:
The four ``if options['something'] is None:`` may be generalized into:

    def set_option(key, options, config):
        options[key] = config[key.upper]

and then replaced by 

    set_option(hq_token, options, config)
    set_option(hq_url, options, config)
    set_option(dispatcher_url, options, config)
    etc...